apis/v1alpha1/shared_types.go

```diff
@@ -211,6 +211,19 @@ type RouteStatus struct {
 	//
 	// +kubebuilder:validation:MaxItems=100
 	Gateways []RouteGatewayStatus `json:"gateways"`
+
+	// AdmittedGateways is a list of Gateways that have admitted this route.
+	// This list includes the namespace and name of each Gateway in a
+	// namespace/name format. This field is primarily used to provide kubectl
+	// output.
+	//
+	// A maximum of 100 Gateways will be represented in this list. If this list
```

Comment:
Can we leave the #'s out of this comment? We should just say that this list may be truncated and also include an indicator for truncation:

`AdmittedGatewaysTruncated bool`

or something similar, maybe 

`AdmittedGatewaysCount int32`

that people can see total even though it was truncated.